docs/source/history.rst

```diff
@@ -5,19 +5,27 @@ Release history
 
 .. towncrier release notes start
 
-Trio_Asyncio 0.8.1 (2018-08-25)
+trio-asyncio 0.8.1 (2018-08-25)
 -------------------------------
 
 Features
 ~~~~~~~~
 
-- `trio_ayncio` now contains an `allow_asyncio` wrapper which allows you to
-  seamlessly mix asyncio and trio semantics:: from trio_asyncio import run
-  allow_asyncio async def main(): print("Sleeping 1") await asyncio.sleep(1)
-  print("Sleeping 2") await trio.sleep(1) run(allow_asyncio, main) The problem
-  with this solution is that Trio's cancellations will no longer be converted
-  to asyncio's `CancelledError` within asyncio code. This may or may not be an
-  issue for your code. (`#30
+- `trio_asyncio` now contains an `allow_asyncio` wrapper which allows you to
+  seamlessly mix asyncio and trio semantics::
+
+      from trio_asyncio import run
+      @allow_asyncio
+      async def main():
+          print("Sleeping 1")
+	  await asyncio.sleep(1)
+          print("Sleeping 2")
+	  await trio.sleep(1)
+      run(allow_asyncio, main)
```

Comment:
Tabs/spaces are mixed here. Also, it would be nice to use a working snippet by fixing imports and removing the decorator:

```python3
import asyncio
import trio
from trio_asyncio import run, allow_asyncio

async def main():
    print("Sleeping 1")
    await asyncio.sleep(1)
    print("Sleeping 2")
    await trio.sleep(1)
run(allow_asyncio, main)
```